lib/models/application.js

```diff
@@ -41,6 +41,9 @@ const getApplicationModel = function (deps, opts) {
 
 	const deviceModel = once(() => require('./device').default(deps, opts));
 	const releaseModel = once(() => require('./release').default(deps, opts));
+	const inviteModel = once(() =>
+		require('./application-invite').default(deps, opts, exports.get),
```

Comment:
I think that this needs to be:
```suggestion
		require('./application-invite').default(deps, opts, (nameOrSlugOrId) => exports.get(nameOrSlugOrId)),
```
since `exports.get` isn't yet defined (and I don't expect the `once` to have any effect, since we are calling this right away).